libs/auth/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@daffodil/auth",
+  "version": "0.0.0-PLACEHOLDER",
+  "description": "Reducers, State, Selectors, and Drivers for logging in and registering users.",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/graycoreio/daffodil"
+  },
+  "author": "Graycore LLC",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/graycoreio/daffodil/issues"
+  },
+  "publishConfig": {
+    "directory": "../../dist/auth"
+  },
+  "scripts": {
+    "build": "ng build auth --prod",
+    "lint": "ng lint auth",
+    "test": "ng test auth --watch=false --browsers=ChromeHeadless",
+    "publish": "(cd ../../dist/auth; npm publish --access=public)"
+  },
+  "homepage": "https://github.com/graycoreio/daffodil",
+  "peerDependencies": {
```

Comment:
Most of these deps aren't real yet, and should not be listed here.

Reply:
Which ones can be removed? Should they be listed somewhere else?

Reply:
Oh do you mean only add them to `package.json` once they are actually needed?